src/python/foglamp/core/routes.py

```diff
@@ -62,6 +64,20 @@ def setup(app):
     app.router.add_route('GET', '/foglamp/audit/logcode', api_audit.get_audit_log_codes)
     app.router.add_route('GET', '/foglamp/audit/severity', api_audit.get_audit_log_severity)
 
+    # Micro Service support - Core
+    app.router.add_route('GET', '/foglamp/service/ping', service_registry.ping)
+
+    app.router.add_route('POST', '/foglamp/service', service_registry.register)
+    app.router.add_route('DELETE', '/foglamp/service/{service_id}', service_registry.unregister)
+    app.router.add_route('GET', '/foglamp/service', service_registry.get_service)
+    app.router.add_route('GET', '/foglamp/service/{service_id}', service_registry.get_service)
```

Comment:
instead of id,  need to add by type or name ; `/foglamp/service/{service_type}` and `/foglamp/service/{service_name}`